README.md

```diff
@@ -100,6 +100,7 @@ cd sgx-lkl
 
 3. Install the Open Enclave build dependencies:
 ```
+git submodule update --recursive --progress --init openenclave
```

Comment:
Wouldn't it be easier to just add `--recursive` to `git clone`?

Reply:
Done